spec/forms/steps/cost/case_type_show_more_form_spec.rb

```diff
@@ -15,6 +15,41 @@
       allow(Steps::Cost::CaseTypeForm).to receive(:choices).and_return([:bingo_duty])
       expect(described_class.choices).to_not include(:bingo_duty)
     end
+
+    it 'returns the relevant choices' do
+      expect(described_class.choices).to eq(%w(
```

Comment:
Is `eq` here going to prove to be brittle with an array this length, or is the ordering of these mission-critical?  Would `match_array`, which doesn't care about ordering, be a better choice here?